.github/workflows/e2e-test-linux-vm.yaml

```diff
@@ -23,7 +23,7 @@ env:
   TARGET_REPO: arc_e2e_test_dummy
   IMAGE_NAME: "arc-test-image"
   IMAGE_VERSION: "dev"
-  
+
```

Comment:
I've been thinking, sometimes we will want to run a single test but not the others. Can we add for each test boolean input that is "True" by default? This way, if we want to have a custom run with a subset of the tests, we have the option to do it

Reply:
These tests are running in parallel and they are all critical scenarios for ARC, do you have any case that we only want to run part of it?

Reply:
Correct and for PRs that option will not be available, they'll all run by default. The inputs will be handy If this e2e workflow is triggered manually. What if we want to troubleshoot a test? Like the Kubernetes mode tests for example, it'll be handy to be able to trigger only one of the bunch

Reply:
I want to avoid extra inputs and complicate `if` conditions to make the workflow more unreadable.
Since these jobs do not depend on each other, those extra jobs won't affect your ability to debug a certain job.